drivers/include/ad7746.h

```diff
@@ -0,0 +1,279 @@
+
+/**
+ * @brief   Convenience function that reads a raw value from the capacitance
+ *          channel
+ *
+ * @param[in] dev   device descriptor
+ * @param[out] raw  read value
+ *
+ * @return AD7746_OK on success
+ * @return AD7746_NODATA if no data is available
+ * @return AD7746_I2C on error getting a reponse
+ */
+static inline int ad7746_read_raw_cap_ch(const ad7746_t *dev, uint32_t *raw)
+{
+    return ad7746_read_raw_ch(dev, AD7746_READ_CAP_CH, raw);
+}
+
+/**
+ * @brief   Convenience function that reads a raw value from the voltage /
+ *          temperature channel
+ *
+ * @param[in] dev   device descriptor
+ * @param[out] raw  read value
+ *
+ * @return AD7746_OK on success
+ * @return AD7746_NODATA if no data is available
+ * @return AD7746_I2C on error getting a reponse
+ */
+static inline int ad7746_read_raw_vt_ch(const ad7746_t *dev, uint32_t *raw)
+{
+    return ad7746_read_raw_ch(dev, AD7746_READ_VT_CH, raw);
+}
+
+/**
+ * @brief   Converts a raw code into a capacitance expressed in fF
+ *
+ * @param[in] raw   Raw code from the device
+ *
+ * @return capacitance in fF
+ */
+int ad7746_raw_to_capacitance(uint32_t raw);
+
+/**
+ * @brief   Converts a raw code into temperature expressed in celsius (C)
+ *
+ * @param[in] raw   Raw code from the device
+ *
+ * @return temperature in celsius
+ */
+int ad7746_raw_to_temperature(uint32_t raw);
+
+/**
+ * @brief   Converts a raw code into voltage expressed in mV
+ *
+ * @param[in] raw   Raw code from the device
+ * @return voltage in mV
+ *
+ * @note Note when using the @ref AD7746_VT_MD_VDD mode, that the voltage from the
+ *       VDD pin is internally attenuated by 6
+ */
+int ad7746_raw_to_voltage(uint32_t raw);
+
+/**
+ * @brief   Sets the current input for the capacitive measurement
+ *
+ * @param[in] dev      device descriptor
+ * @param[in] input  selected input - 0 for CIN1, 1 for CIN2
+ *
+ * @return AD7746_OK on success
+ * @return AD7746_NOI2C on error
+ */
+int ad7746_set_cap_ch_input(const ad7746_t *dev, uint8_t input);
+
+/**
+ * @brief   Sets the mode for the voltage / temperature channel
+ *
+ * @param[in] dev   device descriptor
+ * @param[in] mode  mode to which the channel has to be set
+ *
+ * @return AD7746_OK on success
+ * @return AD7746_NOI2C on error
+ */
+int ad7746_set_vt_channel_mode(const ad7746_t *dev, ad7746_vt_mode_t mode);
+
+/**
+ * @brief   Sets the sample rate for the voltage / temperature channel
+ *
+ * @param[in] dev   device descriptor
+ * @param[in] sr    sample rate
+ *
+ * @return AD7746_OK on success
+ * @return AD7746_NOI2C on error
+ */
+int ad7746_set_vt_sr(const ad7746_t *dev, ad7746_vt_sample_rate_t sr);
+
+/**
+ * @brief   Sets the sample rate for the capacitance channel
+ *
+ * @param[in] dev   device descriptor
+ * @param[in] sr    sample rate
+ *
+ * @return AD7746_OK on success
+ * @return AD7746_NOI2C on error
+ */
+int ad7746_set_cap_sr(const ad7746_t *dev, ad7746_cap_sample_rate_t sr);
```

Comment:
I really don't know anything about the device (yet) but I'm wondering if there's need for us to maintain different sample rates for both sensors. What can we expect, energy savings, better resolution?